pkg/etcdenvvar/etcd_env.go

```diff
@@ -304,6 +308,33 @@ func getCipherSuites(envVarContext envVarContext) (map[string]string, error) {
 	}, nil
 }
 
+func getTLSMinVersion(envVarContext envVarContext) (map[string]string, error) {
```

Comment:
given your extraordinary gotpl skills, I'm wondering whether we should just check this inside of the cipher suite function above in conjunction.

That way we can enable/disable the suites depending on the TLS version. I'm mostly curious about how we can surface this back to a user somehow. I know some (especially FIPS) customers run compliance scanners that will alert them on not setting those ciphers.

Reply:
Knowing that the setting is ignored, even if we were to pass it along, I agree.
We could output a `CIPER_SUITES` env var when tls min version == "TLS1.3"?
I would say that a scanner would have to evolve to know that using TLS1.3 sets the ciphers automatically.
I think enabling FIPS also limits the TLS1.3 ciphers. We'll have to confirm that we output the correct list.